js/zoomViewer.js

```diff
@@ -0,0 +1,24 @@
+let zoomViewer
+
+export default map => {
+  if (zoomViewer != null) return zoomViewer
```

Comment:
Honestly I'm not sure if I should have it return an instance of the Zoom Viewer (with it already added to the map), which I've done here, or if I should return the constructor.

Reply:
Based on https://leafletjs.com/examples/extending/extending-3-controls.html and https://github.com/Leaflet/Leaflet.fullscreen/blob/gh-pages/dist/Leaflet.fullscreen.js, I think the pattern for leaflet controls should be to import the leaflet object and attach things to it. In this case, we'd modify `L.Control` and `L.control`.